index.md

```diff
@@ -3,7 +3,7 @@ title: Developer Portal
 layout: front-page
 front_page:
   ingress: |
-We've compiled a complete toolbox for integrating our payment solutions, so you can familiarize yourself with their various features and functionalities.
+    We've compiled a complete toolbox for integrating our payment solutions, so you can familiarize yourself with their various features and functionalities.
```

Comment:
@arebra This indentation was the critical thing changed in #1669 that broke the front-page. :)

Reply:
The grain of sand that halted the clockwork :)